MyCoolApi.Tests/OSTests.cs

```diff
@@ -8,6 +8,13 @@ namespace MyCoolApi.Tests;
 [TestClass]
 public class OSTests
 {
+    [TestMethod]
+    // FYI, this is a newly added attribute that inherits from the customizable ConditionBaseAttribute.
+    [OSCondition(OperatingSystems.Linux)]
```

Comment:
Should this "skip" the test if the condition isn't met? because this is still reporting as a failure on windows. I guess I expect that it would use this as a condidion to run and report skipped rather than fail.

Reply:
Interesting it was correctly skipped for me in VS . The attribute offers 2 modes: include (default) and exclude so that you can list where to run or where not to run.